### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by dvvanessastoiber**
## Release notes

* Update README.md (datavisyn/tdp_matomo#31)
* Update author name (datavisyn/tdp_matomo#29)
* Update README.md circleci links & logo (datavisyn/tdp_matomo#30)
* Migrate to jest for testing (datavisyn/tdp_matomo#15)
* Refactoring

### Release dependencies first

In case of dependent Phovea/TDP repositories follow [dependency tree](https://wiki.datavisyn.io/phovea/fundamentals/development-process#dependency-hierarchy) from the top:

 
### 🏁 Finish line

* [ ] Inform colleagues and customers about the release
* [ ] Celebrate the new release 🥳

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,42 +73,19 @@
   },
   "dependencies": {
     "crypto-js": "^4.0.0",
-    "tdp_core": "github:datavisyn/tdp_core#semver:^9.0.0"
+    "tdp_core": "github:datavisyn/tdp_core#semver:^10.0.1"
```
Comment (thinkh) [inline package.json:76]:
```suggestion
    "tdp_core": "^10.0.1"
```

File: requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
--e git+https://github.com/datavisyn/tdp_core.git@v9.0.0#egg=tdp_core
+-e git+https://github.com/datavisyn/tdp_core.git@v10.0.1#egg=tdp_core
```
Comment (thinkh) [inline requirements.txt:1]:
```suggestion
tdp_core>=10.0.0,<11.0.0
```

File: phovea_registry.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
  * Licensed under the new BSD license, available at http://caleydo.org/license
  **************************************************************************** */
 
-import {register} from 'phovea_core/src/plugin';
+import {PluginRegistry} from 'phovea_core';
 import reg from './src/phovea';
```
Comment (thinkh) [inline phovea_registry.js:8]: Is this correct now?

```suggestion
import reg from './dist/phovea';
```

Comment (dvvanessastoiber) [reply phovea_registry.js:8]: Yes, this should be correct.

**Pull request merged**